docs/application-connector/08-11-rotate-root-ca.md

```diff
@@ -10,7 +10,9 @@ Two different components use the Root CA certificate. As a result, the certifica
   - The `connector-service-app-ca` Connector Service CA Secret responsible for signing certificate requests
```

Comment:
This is just a general note. Users need to bother with root CA creation only if they manage the cluster. In Compass scenario the root CA certificate is managed by the Compass Runtime Agent and no manual rotation is needed.